doc/usage/quickstart.rst

```diff
@@ -145,6 +149,8 @@ an argument to see which targets are available.
    toolchain for you.
 
 
+.. todo:: Move this whole section into a guide on rST or directives
```

Comment:
I have all these `todo`\'s resolved in follow-up patches. I've submitted this separately to keep things easy to review

Reply:
Reasonable, but this todo is useful only for us. What about comment it out?

Reply:
I don't think it matters. I'll be fixing this literally within days. Can we just keep this as is?